lte/gateway/c/oai/lib/pcef/pcef_handlers.cpp

```diff
@@ -57,8 +57,15 @@ static void create_session_response(
 }
 
 static void pcef_fill_create_session_req(
+    std::string& imsi, std::string& ip, ebi_t eps_bearer_id,
     const struct pcef_create_session_data* session_data,
     magma::LocalCreateSessionRequest* sreq) {
+  // Will remove once we migrate all fields to be handled by
```

Comment:
same here. I think `TODO:` highlights in different color in any IDE so it helps to recognize live comments

Reply:
maybe you want to separate the once you want to keep from the ones you want to remove. So next time you do it you dont have to do the effort of backtracking what you did